server/index.js

```diff
@@ -18,6 +18,13 @@ nextApp.prepare().then(() => {
   /* APIs go there */
   expressServer.use('/api', api);
 
+  /*Clean URL for rendering Product Information*/
```

Comment:
space after *. Add [spaced-comment](https://eslint.org/docs/rules/spaced-comment) rule to throw lint errors for the same.